### PR TITLE
[NETBEANS-5027] PHP - improved display of array parameter

### PR DESCRIPTION
**Pull request by KacerCZ**
https://issues.apache.org/jira/browse/NETBEANS-5027

Improves display of array used as default parameter value.
Uses short array format, displays first item in array, indicates more elements in array.

Before:
netbeans5027-before

After:
netbeans5027-after

### CODE REVIEW

**Review by junichi11 — APPROVED** [review]

**Comment by tmysik** [comment]
@KacerCZ 

Could you please check the failed build? Thanks.

File: php/php.editor/test/unit/src/org/netbeans/modules/php/editor/CodeUtilsTest.java
Comment (tmysik) [inline php/php.editor/test/unit/src/org/netbeans/modules/php/editor/CodeUtilsTest.java:224]: Could you please add a test where the array value is an object? I would like to see how the tooltip will look like in such a case. Thanks!

**Review by tmysik — CHANGES_REQUESTED** [review]
Please, see my comment. Thanks.

File: php/php.editor/test/unit/src/org/netbeans/modules/php/editor/CodeUtilsTest.java
Comment (KacerCZ) [reply php/php.editor/test/unit/src/org/netbeans/modules/php/editor/CodeUtilsTest.java:224]: If I understand you correctly you mean this case?
<code>
function printDate(\DateTime $date = new \DateTime()) {}
</code>

PHP does not support object as default value.

**Comment by KacerCZ** [comment]
@tmysik Build failure is related to network issue during build setup. Could you please trigger build again?

**Comment by junichi11** [comment]
@hectorespert Could you have a look at the following?
```
Error: Unable to process command '::add-path::/home/runner/.composer/vendor/bin' successfully.
Error: The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

File: php/php.editor/test/unit/src/org/netbeans/modules/php/editor/CodeUtilsTest.java
Comment (tmysik) [reply php/php.editor/test/unit/src/org/netbeans/modules/php/editor/CodeUtilsTest.java:224]: No. I meant his the tooltip would look like for array e.g. `[0 => new DateTime()]`. Just to verify that it still looks ok. Thanks and sorry for not being clear.

Comment (tmysik) [reply php/php.editor/test/unit/src/org/netbeans/modules/php/editor/CodeUtilsTest.java:224]: Ha, sorry, I was too fast, now I see it. It is only about the parameters, not all arrays "everywhere".

**Review by tmysik — APPROVED** [review]

**Comment by KacerCZ** [comment]
@junichi11 This issue was fixed by @hectorespert in PR #2543

**Comment by junichi11** [comment]
I know that. It occurred again when I reran jobs. (See timestamp of build job.)

**Comment by tmysik** [comment]
So, I would merge it. Any objections?

**Comment by junichi11** [comment]
@tmysik Just in case, let's wait for a reply from @hectorespert .
I reran the jobs after that change was merged into the master branch. That should pass if there is no problem, I suppose.

**Comment by hectorespert** [comment]
For me it's ok, linux tests are failing due the fix was not applied in this PR, but the job to run php tests in windows pass this change.

**Pull request merged**
